diagnostics-app/src/app/layout/side-nav/side-nav.component.html

```diff
@@ -5,13 +5,14 @@
 -->
 
 <div *ngFor="let navList of navLists">
-  <div *ngIf="navList.hasHeader" mat-subheader>{{ navList.header }}</div>
-  <mat-nav-list>
-    <app-side-nav-item
-      *ngFor="let navItem of navList.items"
-      [icon]="navItem.icon"
-      [link]="navItem.link"
-      [name]="navItem.name"
-    ></app-side-nav-item>
-  </mat-nav-list>
+  <div mat-subheader>
+    <div
```

Comment:
My understanding is that we are deprecating the use of `app-side-nav-item`? Instead we are just adding button and links through router. The styling is done by assigning the class name to the div.

If that's the case, I think the `app-side-nav-item` class is now also deprecated, maybe we can delete that class as well?

Reply:
Done.

Reply:
Hmmm from the `file changed` tab of this CL, I don't see the file change for deleting app-side-nav.

Can you verify that the folder/components are indeed destroyed?

Reply:
I've pushed another commit for this.